spec/features/choice_selector_spec.rb

```diff
@@ -0,0 +1,31 @@
+feature 'choice_selector' do
+  scenario 'choices are displayed' do
+    enter_name_and_play
+    click_button 'Start'
+    expect(page).to have_content "Rock, Paper, or Scissors?\nMake your choice."
+  end
+
+  scenario 'player can select rock' do
+    enter_name_and_play
+    click_button 'Start'
+    choose 'Rock', allow_label_click: true
```

Comment:
I hadn't seen this allow_label_click: true before. Such a useful way of testing a specific outcome.